src/pages/Checklist/index.tsx

```diff
@@ -246,80 +286,137 @@ export const Checklist = () => {
+                  }}
+                  description="{code} values are terminal outputs to look for, should not translate"
+                />
               </Text>
             </li>
             <li className="py5">
               <Text>
-                Check if <Code>Local time</Code>, <Code>Time zone</Code>, and{' '}
-                <Code>Universal time</Code> are all correct.
+                <FormattedMessage
+                  defaultMessage="Check if {code1}, {code2}, and {code3} are all correct."
+                  values={{
+                    code1: <Code>Local time</Code>,
+                    code2: <Code>Time zone</Code>,
+                    code3: <Code>Universal time</Code>,
+                  }}
+                  description="{code} values are terminal outputs to look for to confirm system time"
+                />
               </Text>
             </li>
             <li className="py5">
               <Text>
-                If <Code>NTP Service</Code> is not <Code>active</Code>, run
-                <CodeSnippet>
-                  <code>sudo timedatectl set-ntp on</code>
-                </CodeSnippet>
+                <FormattedMessage
+                  defaultMessage="If {code1} is not {code2}, run {code3}"
+                  values={{
+                    code1: <Code>NTP Service</Code>,
+                    code2: <Code>active</Code>,
+                    code3: (
+                      <CodeSnippet>
+                        <code>sudo timedatectl set-ntp on</code>
+                      </CodeSnippet>
+                    ),
+                  }}
+                  description="{code} values are terminal outputs and commands."
+                />
               </Text>
             </li>
             <li className="py5">
               <Text>
-                If you see error message{' '}
-                <Code>Failed to set ntp: NTP not supported</Code>, you may need
-                to install <Code>chrony</Code> or <Code>ntp</Code> package.
+                <FormattedMessage
+                  defaultMessage="If you see error message {code1} , you may need to install {code2} or {code3} package."
+                  values={{
+                    code1: <Code>Failed to set ntp: NTP not supported</Code>,
+                    code2: <Code>chrony</Code>,
+                    code3: <Code>ntp</Code>,
+                  }}
+                  description="{code} values are terminal outputs and commands."
+                />
               </Text>
             </li>
             <li className="py5">
               <Text>
-                <strong>Note:</strong> by default, VMs may disable NTP so you
-                may need to find a work-around for your environment.
+                <FormattedMessage
+                  defaultMessage="{note}: by default, VMs may disable NTP so you may need to find a work-around for your environment."
+                  values={{
+                    note: <strong>Note</strong>,
```

Comment:
This is tricky & something we may want to choose to just remove the bold on "Note".

If we want to keep the bold, we'd still want to translate "Note", so the value should be:
```
note: <strong>formatMessage({defaultMessage: "Note"})</strong>
```
But that would take "Note" out of context. Translators may thinks it's the noun vs. the verb.

Reply:
We could use a `description` to explain it's the verb but the lack of tense general obscurity might make it too confusing to translators for this to be worth it.

Reply:
Yeah, the description was how I was attempting to go about this. Removing the bold would likely be the best option. These aren't really warnings, and don't demand a high level of attention. 
And good call on using `formatMessage()` to translate within the `values` attribute... wasn't thinking, I can add another commit to fix any of these.

Reply:
I agree, in general I suspect the value of bolding words won't be worth the effort/potential confusion. I think this feature will be used best with variables that we do NOT want translated (such as code snippets / CLI commands).